common/src/main/java/com/wynntils/wc/custom/item/properties/SkillPointProperty.java

```diff
@@ -15,7 +15,9 @@
 import net.minecraft.ChatFormatting;
 
 public class SkillPointProperty extends CustomStackCountProperty {
-    private static final Pattern POINT_PATTERN = Pattern.compile("^.+?(\\d+) points .+$");
+    private static final Pattern POINT_PATTERN = Pattern.compile("^§7    (\\d+) points      §r§6\\d+ points$");
```

Comment:
This pattern won't match on HB, the whitespace has changed iirc

Reply:
@DonkeyBlaster but the skill point pattern is still used for other features

Reply:
Ah okay